reana_server/reana_admin.py

```diff
@@ -327,8 +332,17 @@ def _format_statuses(type_, statuses):
             click.echo(f'{stat_name}: {stat_value}')
 
     types = STATUS_OBJECT_TYPES.keys() if 'all' in types else types
+    status_report_output = ''
     for type_ in types:
         statuses_obj = STATUS_OBJECT_TYPES[type_]()
         statuses = statuses_obj.get_status()
-        _format_statuses(type_, statuses)
-        click.echo('\n')
+        status_report_output += _format_statuses(type_, statuses) + '\n'
+
+    if email:
+        if REANA_URL:
```

Comment:
What about `if email and REANA_URL`? Otherwise if `REANA_URL` is None I think it's going to fail as `status_report_body` wouldn't exist.